src/targetPacks/ILsrc/microsoft.aspnetcore.app.ref/3.1.10/[Content_Types].xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Trim this file (multiple projects) `[Content_Types].xml` and:
`_rels/.rels`
`package/services/metadata/core-properties/6d80a3452fce4876a8220ed215d32203.psmdcp`?

Reply:
Removed from all Microsoft.AspNetCore.App.Ref projects.